images/dcos/bootstrap.sh

```diff
@@ -71,11 +71,6 @@ controller_manager_host=${CONTROLLER_MANAGER_HOST:-${default_dns_name}}
 controller_manager_port=${CONTROLLER_MANAGER_PORT:-10252}
 echo "* controller manager: $controller_manager_host:$controller_manager_port"
 
-# assume that the leading mesos master is always running a marathon
-# service proxy, perhaps using haproxy.
-service_proxy=${SERVICE_PROXY:-leader.mesos}
```

Comment:
I could not find a reference to it, and @jdef thought it may be safe to remove.  Do you know otherwise?